index.html

```diff
@@ -28,6 +28,131 @@ <h1><span class="greetings">Hey there. I’m Josue</span><br>
+            <li class="language">JavScript</li>
+          </ul>
+          <button class="button" type="button">See Project</button></span>
+      </article>
+      <article class="project-template">
+        <span class="place-holder">
+          <h3>Multi-Post Stories <br>Gain+Glory</h3>
+          <ul class="pr-language">
+            <li class="language">Ruby on rails</li>
+            <li class="language">css</li>
+            <li class="language">html</li>
+            <li class="language">JavScript</li>
+          </ul>
+          <button class="button" type="button">See Project</button></span>
+      </article>
+      <article class="project-template">
+        <span class="place-holder">
+          <h3>Multi-Post Stories <br>Gain+Glory</h3>
+          <ul class="pr-language">
+            <li class="language">Ruby on rails</li>
+            <li class="language">css</li>
+            <li class="language">html</li>
+            <li class="language">JavScript</li>
+          </ul>
+          <button class="button" type="button">See Project</button></span>
+      </article>
+      <article class="project-template">
+        <span class="place-holder">
+          <h3>Multi-Post Stories <br>Gain+Glory</h3>
+          <ul class="pr-language">
+            <li class="language">Ruby on rails</li>
+            <li class="language">css</li>
+            <li class="language">html</li>
+            <li class="language">JavScript</li>
+          </ul>
+          <button class="button" type="button">See Project</button></span>
+      </article>
+      <article class="project-template">
+        <span class="place-holder">
+          <h3>Multi-Post Stories <br>Gain+Glory</h3>
+          <ul class="pr-language">
+            <li class="language">Ruby on rails</li>
+            <li class="language">css</li>
+            <li class="language">html</li>
+            <li class="language">JavScript</li>
+          </ul>
+          <button class="button" type="button">See Project</button></span>
+      </article>
+             <!-- About me project part -->
+    <h2 class="titles-about-me">About me</h2>
+    <p class="paragraph">
+      I can help you build a product , feature or website Look through
+      some of my work and experience! If you like what you see and have a
+      project you need coded, don’t hestiate to contact me.
+    </p>
+    <button class="button-resume" type="button">get my resume</button>
+    <article class="resume-card">
+      <img
+        class="resume-img"
+        src="./tools/icon-languages.svg"
+        alt="language-image"
+      />
+      <h3 class="blue">Languages</h3>
+      <ul class="about-language">
+        <li class="green ">JavaScript</li>
+        <li class="green ">Ruby</li>
+        <li class="green ">Html</li>
+        <li class="green ">CSS</li>
+      </ul>
+    </article>
+    <article class="resume-card">
+      <img
+        class="resume-img"
+        src="./tools/icon-frameworks.svg"
```

Comment:
- [ ] Since this describes a list of technology categories (Languages, Frameworks, Skills)📜. It would be a great idea to use a `<ul>` tag wrapping those containers (and each one of them inside a `<li>` tag). This would improve the semantics of your project. Please, change it! 🛠️

Reply:
used <p> tag in my program flow

Reply:
used the "<p>" tag in my HTML flow.